frontend/src/metabase/actions/components/ActionCreator/FormCreator/utils.ts

```diff
@@ -230,6 +144,9 @@ export const getInputType = (param: Parameter, field?: Field) => {
   if (field.isDate()) {
     return field.isDateWithoutTime() ? "date" : "datetime";
   }
+  if (field.semantic_type === TYPE.Email) {
+    return "string";
+  }
```

Comment:
This makes me think: don't we have an `email` input type? 🤔 
Also nit: maybe it'd be nicer to combine this with [this check](https://github.com/metabase/metabase/blob/98182eab7179f8d24acc06b75f4620b81c6d4028/frontend/src/metabase/actions/components/ActionCreator/FormCreator/utils.ts#L157) below

Reply:
there is an email input component, but we don't allow users to set email fields in the form creator, so i was trying to align this with the available options.